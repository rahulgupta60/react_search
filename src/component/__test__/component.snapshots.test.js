import React from 'react';
import TestRenderer from 'react-test-renderer'; // ES6

import { GenreList, MovieList, RatingList } from '../index';

describe('Snapshot', () => {
  test('GenreList', () => {
    const component = TestRenderer.create(
      <GenreList genres={[]} genresSelected={[]} onClick={() => {}} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('MovieList', () => {
    const component = TestRenderer.create(
      <MovieList genres={[]} movies={[]} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('RatingList', () => {
    const component = TestRenderer.create(
      <RatingList ratingList={[]} value={1} onChange={() => {}} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
