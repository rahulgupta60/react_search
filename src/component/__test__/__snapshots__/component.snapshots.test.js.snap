// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot GenreList 1`] = `
<div
  className="genres-list"
>
  <div
    className="form-group"
  >
    <label
      className="control-label"
    >
      Filter by genre :
    </label>
    <div
      className="text-uppercase text-warning well well-sm"
    >
      by default 
      <strong>
        no
      </strong>
       filter apply for the genres 
    </div>
  </div>
</div>
`;

exports[`Snapshot MovieList 1`] = `"No Record found"`;

exports[`Snapshot RatingList 1`] = `
<div
  className="form-group"
>
  <label
    className="control-label"
  >
    Filter by rating:
  </label>
  <select
    className="form-control"
    onChange={[Function]}
    placeholder="select"
    value={1}
  >
    ;
  </select>
</div>
`;
